test/async-hooks/test-graph.signal.js

```diff
@@ -1,6 +1,11 @@
 'use strict';
```

Comment:
Most of the async hooks can be represented as a graph/tree (see dprof.js.org). The `test-graph.*.js` files check that the graph is correct. The `test-graph.signal.js` file check that the graph regarding signals is correct.